roles/kubernetes/kubeadm/tasks/main.yml

```diff
@@ -80,10 +83,16 @@
     | {{ bin_dir }}/kubectl replace -f -
   delegate_to: "{{groups['kube-master']|first}}"
   run_once: true
-  when: is_kube_master and kubeadm_discovery_address != kube_apiserver_endpoint
+  when:
+    - kubeadm_config_api_fqdn is not defined
```

Comment:
Question, this just skip updating the `server` field if `kubeadm_config_api_fqdn` defined so it would not work if you already setted up the cluster with kubeadm and no LB and later on you plan to add the LB, right?

Reply:
no 
if kubeadm_config_api_fqdn is not defined, use local nginx LB.
else if kubeadm_config_api_fqdn is defined, it will confiure LB by kubeadm.

Reply:
If I deploy a cluster not using LB and later on I add it, kubeadm will update the config? Thought that only happen on `kubeadm init`.